src/SDK/Subscriber/Listener/EndSpanListener.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
these example listeners shouldn't be in the SDK (unless you plan to expand on them to provide built-in functionality).